source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Provider Physical Network
+category: feature
+authors: phoracek,amusil
+feature_name: Provider Physical Network
+feature_modules: engine,vdsm,ovn-provider
+feature_status: In Development
+---
+
+# Provider Physical Network
+
+## Summary
+
+This feature extends [ovirt-ovn-provider](https://www.ovirt.org/develop/release-management/features/network/ovirt-ovn-provider/) adding support for physical network access. It allows user to connect external OVN networks to a physical (oVirt native) network.
+
+Feature progress is tracked on [ovn-localnet Trello](https://trello.com/b/lxIBEn0A/ovn-localnet). Related patches can be found on [gerrit topic:localnet](https://gerrit.ovirt.org/#/q/topic:localnet).
+
+### Owner
+
+- Feature Owner: Petr Horáček (phoracek)
+- E-mail: <phoracek@redhat.com>
+
+### Benefit to oVirt
+
+Existing OVS VM networking could be replaced with OVN. That would also allow us to drop a part of VM live migration hook.
+
+Physical network access will be required once NAT and routing are introduced to the OVN provider.
+
+## Usage
+
+### Set cluster network switch type to OVS
+
+This feature requires cluster switch type set to OVS. You can enable OVS networking using the following steps.
+
+1) **Enable OVS hook for live migration on all vdsm hosts.** Add `migration_ovs_hook_enabled = true` to `[vars]` section in `/etc/vdsm/vdsm.conf`. Restart vdsm with `systemctl restart vdsmd supervdsmd`.
```

Comment:
why is this required? I think that with OVN-only networks it is not needed at all.

Reply:
For physical networks it is needed. We need an OVS bridge to connect to.